<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.ico</normaloff>logo.ico</iconset>
  </property>
  <widget class="QTableWidget" name="skureview_tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>561</width>
     <height>651</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhLatinOnly</set>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AllEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>100</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>250</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>250</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="skureview_pushButton_generateCSV">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>830</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QLabel" name="skureview_label_warehosue">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Warehouse:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="skureview_textEdit_warehouse">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="skureview_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;New Order&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Inventory Review</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="skureview_label_demo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="skureview_pushButton_add50Lines">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>780</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add extra 50 lines</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
